archetypes/database-se/src/main/resources/src/main/java/__pkg__/Main.java.mustache

```diff
@@ -0,0 +1,96 @@
+
+package {{package}};
+
+import java.io.IOException;
+import java.util.logging.LogManager;
+
+import io.helidon.config.Config;
+import io.helidon.dbclient.DbClient;
+import io.helidon.dbclient.health.DbClientHealthCheck;
+import io.helidon.health.HealthSupport;
+import io.helidon.media.jsonb.JsonbSupport;
+import io.helidon.media.jsonp.JsonpSupport;
+import io.helidon.metrics.MetricsSupport;
+import io.helidon.webserver.Routing;
+import io.helidon.webserver.WebServer;
+
+/**
+ * Main class that starts up server and registers Pokemon services.
+ */
+public final class Main {
+
+    /**
+     * Cannot be instantiated.
+     */
+    private Main() {
+    }
+
+    /**
+     * Application main entry point.
+     *
+     * @param args command line arguments
+     * @throws java.io.IOException if there are problems reading logging properties
+     */
+    public static void main(final String[] args) throws IOException {
+        startServer();
+    }
+
+    /**
+     * Start the server.
+     *
+     * @return the created {@link io.helidon.webserver.WebServer} instance
+     * @throws java.io.IOException if there are problems reading logging properties
+     */
+    static WebServer startServer() throws IOException {
+        // Load logging configuration
+        LogManager.getLogManager().readConfiguration(Main.class.getResourceAsStream("/logging.properties"));
+
+        // By default this will pick up application.yaml from the classpath
+        Config config = Config.create();
+
+        // Prepare routing and build server
+        Routing routing = createRouting(config);
+        WebServer server = WebServer.builder(routing)
+                .addMediaSupport(JsonpSupport.create())
+                .addMediaSupport(JsonbSupport.create())
+                .config(config.get("server"))
+                .build();
+
+        // Start the server and print some info
+        server.start()
+                .thenAccept(ws -> System.out.println("WEB server is up! http://localhost:" + ws.port() + "/"));
+
+        // Server threads are not daemon
+        server.whenShutdown()
+                .thenRun(() -> System.out.println("WEB server is DOWN. Good bye!"));
+
```

Comment:
Other archetypes seem to use a different pattern:
https://github.com/oracle/helidon/blob/6e4e3aa3e749c9280eea06f927004cb7e4cfd940/archetypes/bare-se/src/main/resources/src/main/java/__pkg__/Main.java.mustache#L55-L68

We should probably be consistent unless it makes sense otherwise.